internal/cryptokit/hash.go

```diff
@@ -170,7 +170,7 @@ func (h *evpHash) WriteString(s string) (int, error) {
 }
 
 func (h *evpHash) WriteByte(c byte) error {
-	h.writeFunc(h.ptr, base([]byte{c}), C.int(1))
+	h.writeFunc(h.ptr, (*C.uchar)(unsafe.Pointer(&c)), 1)
```

Comment:
It's not clear to me from the PR description or commits what this change from `base(...)` is for.